packages/ipfs-http-client/package.json

```diff
@@ -18,9 +18,7 @@
   "types": "./dist/src/index.d.ts",
   "browser": {
     "./src/lib/multipart-request.js": "./src/lib/multipart-request.browser.js",
-    "ipfs-utils/src/files/glob-source": false,
```

Comment:
This was moved to https://github.com/ipfs/js-ipfs-utils/pull/121/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R18